modules/manage/partials/monitor-health.adoc

```diff
@@ -209,19 +209,137 @@ Leaderless partitions can be caused by unresponsive brokers. When an alert on `r
+          - consumer_lag
+----
+
+```bash
+kubectl apply -f redpanda-cluster.yaml --namespace <namespace>
+```
+
+--
+Helm::
++
+--
+[tabs]
+====
+--values::
++
+.`enable-consumer-metrics.yaml`
+[,yaml]
+----
+config:
+  cluster:
+    enable_consumer_group_metrics:
+      - group
+      - partition
+      - consumer_lag
+----
++
+```bash
+helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
+--values enable-consumer-metrics.yaml --reuse-values
+```
+
+--set::
++
+[,bash]
+----
+helm upgrade --install redpanda redpanda/redpanda \
+  --namespace <namespace> \
+  --create-namespace \
+  --set config.cluster.enable_consumer_group_metrics[0]=group \
+  --set config.cluster.enable_consumer_group_metrics[1]=partition \
+  --set config.cluster.enable_consumer_group_metrics[2]=consumer_lag
+----
+
+====
+--
+======
+endif::[]
+
+When these properties are enabled, Redpanda computes and exposes the `redpanda_kafka_consumer_group_lag_max` and `redpanda_kafka_consumer_group_lag_sum` gauges in the public metrics endpoint.
+
+Enabling these additional metrics may add extra processing overhead to the broker, especially in environments with a very high number of consumer groups or partitions.
+The increased frequency of metric collection could result in higher resource utilization.
+If your environment is sensitive to performance overhead or if you prefer to avoid any additional load, you may use the <<offset-based-calculation, offset-based calculation method>>. Otherwise, enabling these properties can simplify monitoring and improve the accuracy of consumer group lag measurements across diverse observability platforms.
+
+==== Offset-based calculation
+
+If you prefer not to enable the <<dedicated-gauges, dedicated gauges>>, you can calculate consumer group lag using offset metrics.
+
+Redpanda provides two metrics to calculate consumer group lag:
+
+- xref:reference:public-metrics-reference.adoc#redpanda_kafka_max_offset[`redpanda_kafka_max_offset`]: The broker's latest offset for a partition.
+- xref:reference:public-metrics-reference.adoc#redpanda_kafka_consumer_group_committed_offset[`redpanda_kafka_consumer_group_committed_offset`]: The last committed offset for a consumer group on that partition.
+
+For example, here's a typical query to compute consumer lag:
 
 [,promql]
 ----
 max by(redpanda_namespace, redpanda_topic, redpanda_partition)(redpanda_kafka_max_offset{redpanda_namespace="kafka"}) - on(redpanda_topic, redpanda_partition) group_right max by(redpanda_group, redpanda_topic, redpanda_partition)(redpanda_kafka_consumer_group_committed_offset)
 ----
 
+This approach works well if your observability platform supports functions, such as `max()`, on calculated values. However, it may not be suitable for all platforms, as some may not support complex queries or functions. In such cases, consider enabling the <<dedicated gauges, dedicated gauges>> for a simpler and more accurate monitoring experience.
```

Comment:
Seems like you already mentioned that is may not be suitable up above. This paragraph seems to duplicate some of what is said earlier. I'm keen to know the use cases where this is preferable to dedicated.